libs/datasets/region_aggregation.py

```diff
@@ -287,6 +294,10 @@ def _aggregate_dataframe_by_region(
     # Aggregate by location_id_agg, optional date and variable.
     long_agg = long_all_values.groupby(groupby_columns, sort=False).sum()
 
+    long_agg = long_agg.loc[
+        ~long_agg.index.get_level_values(PdFields.VARIABLE).isin(FIELDS_NOT_TO_AGGREGATE)
+    ]
```

Comment:
Without this, we were getting cdc_community_level entries in `data/multiregion-wide-dates.csv` for cbsas that were just summing up all the levels within the counties.